<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogemploye</class>
 <widget class="QDialog" name="Dialogemploye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des employés</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des employés</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>881</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius: 10px;
padding:  0 9px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>311</width>
       <height>511</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #8B0000;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Ajouter un Employé</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
      </property>
      <property name="text">
       <string>N° Téléphone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeId">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeTitre">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeDescription">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Lemail">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAjouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;</string>
      </property>
      <property name="text">
       <string>Ajouter </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  </string>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>172</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;</string>
      </property>
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Journaliste</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>351</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #8B0000;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Modifier Ou Supprimer un employé</string>
     </property>
     <widget class="QPushButton" name="pushButtonModifier">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSupprimer">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 10px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>320</y>
       <width>551</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #8B0000;
  font-size: 16px;</string>
     </property>
     <property name="title">
      <string>Rechercher ou Trier les employés</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
      </property>
      <property name="text">
       <string>Rechercher par CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
      </property>
      <property name="text">
       <string>Rechercher par Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeIdRT">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeTitreRT">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
      </property>
      <property name="text">
       <string>Trier par cin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
      </property>
      <property name="text">
       <string>Trier par nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>109</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: none;
  color: black;
  
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  
</string>
      </property>
      <property name="text">
       <string>Rechercher par Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>111</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 11px;
  margin: 4px 2px;
  border-radius : 16px;</string>
      </property>
      <property name="text">
       <string>Trier par salaire</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Tab2">
    <attribute name="title">
     <string>Génération d'un fichier pdf</string>
    </attribute>
    <widget class="QTextEdit" name="imp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>1241</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius: 10px;
padding:  0 9px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Impdos_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>691</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #8B0000	;
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
font-family:calibri;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  border-radius : 16px;</string>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
